channels/drdynvc/client/drdynvc_main.c

```diff
@@ -1187,6 +1188,7 @@ static void VCAPITYPE drdynvc_virtual_channel_open_event_ex(LPVOID lpUserParam,
 	if (!drdynvc || (drdynvc->OpenHandle != openHandle))
 	{
 		WLog_ERR(TAG, "drdynvc_virtual_channel_open_event: error no match");
+		Stream_Free((wStream*) pData, TRUE);
 		return;
```

Comment:
@akallabeth in fact the leak was here. In this function we have ownership on the stream, so we must release it.